src/dymaptic.GeoBlazor.Core/Components/Layers/Graphic.cs

```diff
@@ -68,6 +72,13 @@ public Graphic()
     [Parameter]
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public AttributesDictionary Attributes { get; set; } = new();
+    
+    /// <summary>
+    ///     Indicates the visibility of the graphic. Default value: true.
+    /// </summary>
+    [Parameter]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public bool? Visible { get; set; }
```

Comment:
so it default to true in JS if its null?

Reply:
Yes, using the `copyValuesIfExist` pattern in the TypeScript, we only set this if the user does, otherwise we leave it unset and the ArcGIS defaults take over.